### ISSUE TITLE
Bug when converting a process tree to a Petri net

### ISSUE THREAD

**Issue by EduardoGoulart1** [process-intelligence-solutions/pm4py#374]
There seems to be a bug when converting a process tree to a Petri net:

```
import pm4py
pt = pm4py.parse_process_tree("*(tau, '4', '5')")
pm4py.view_petri_net(*pm4py.convert_to_petri_net(pt))
```

![tmpjjqz6r_9 gv](https://user-images.githubusercontent.com/67106692/211628368-f7e26be1-ed59-4058-a16c-7fc79e2986ba.png)

I had a short look into the conversion code to find the bug, but I didn't manage to. It feels a bit like that section of code could be simplified.

**Comment by fit-alessandro-berti** [comment]
Dear @EduardoGoulart1 

The Petri net seems to reflect the intended behavior: a loop with an invisible as DO part, a "4" as REDO part, and a "5" as an EXIT part.

The invisibles could be simplified a bit more (for example, the second invisible after the source place) but is not fundamentally wrong

**Comment by EduardoGoulart1** [comment]
> Dear @EduardoGoulart1
> 
> The Petri net seems to reflect the intended behavior: a loop with an invisible as DO part, a "4" as REDO part, and a "5" as an EXIT part.
> 
> The invisibles could be simplified a bit more (for example, the second invisible after the source place) but is not fundamentally wrong

Hi Ale

what exactly is meant by "exit" part? AFAIK `*(A, B, C)` should be equivalent to `*(A, X(B, C))`. At least this is how Sander defined it.

**Comment by fit-alessandro-berti** [comment]
Yes, I checked and you are right. We first introduced in pm4py a process tree operator with DO-REDO-EXIT parts. However, then we switched to a DO-REDO paradigm. With insights on such change, we should now make the conversion of a three-child process tree as you describe.

**Comment by fit-alessandro-berti** [comment]
Dear @EduardoGoulart1 

We fixed the issue in pm4py 2.5.0
